src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,11 +6,11 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix(":name");
-    public static final Prefix PREFIX_PHONE = new Prefix(":phone");
-    public static final Prefix PREFIX_EMAIL = new Prefix(":email");
-    public static final Prefix PREFIX_YEAR_JOINED = new Prefix(":year");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_NAME = new Prefix(":name ");
+    public static final Prefix PREFIX_PHONE = new Prefix(":phone ");
+    public static final Prefix PREFIX_EMAIL = new Prefix(":email ");
+    public static final Prefix PREFIX_YEAR_JOINED = new Prefix(":year ");
+    public static final Prefix PREFIX_ADDRESS = new Prefix(":address ");
+    public static final Prefix PREFIX_TAG = new Prefix(":tag ");
```

Comment:
Are white spaces necessary for Prefixes? 
Perhaps consider to remove the white spaces.
```":name "``` to ```":name"```

Reply:
fixed